SConstruct

```diff
@@ -87,6 +87,7 @@ env_base.split_drivers = False
 env_base.split_modules = False
 env_base.module_version_string = ""
 env_base.msvc = False
+env_base.use_mingw = ARGUMENTS.get("use_mingw", False)
```

Comment:
Why add it to the base env? It's defined as platform-specific option at https://github.com/godotengine/godot/blob/fb29d54dc7301dae422437ad68b0a53c68cc404f/platform/windows/detect.py#L68, so you can just query `env["use_mingw"]`.